content/integrations/statsd.md

```diff
@@ -12,7 +12,9 @@ Enable the Datadog StatsD integration to:
   * Monitor uptime and availability of a StatsD server
   * Collecting metrics from a StatsD server's admin interface
 
-**Please note the Datadog Agent includes [dogstatsd](http://docs.datadoghq.com/guides/dogstatsd/) which serves as a StatsD forwarder. This integration is intended for monitoring external StatsD servers, and is not needed to send metrics to Datadog using the StatsD protocol.**
+**Please note the Datadog Agent includes [dogstatsd](http://docs.datadoghq.com/guides/dogstatsd/) which serves as a StatsD forwarder. If you want to send metrics from a statsd server, just point it to the forwarder listening on port 17123.**
```

Comment:
Also the second bullet point:

> Collecting metrics from a StatsD server's admin interface

*does* make it sound like this will connect to StatsD and pull in all the application metrics it is collecting.

Updated proposal:

> Monitor the availability of your StatsD servers and track how busily they're processing application metrics.
>
> This Agent check does NOT help you forward application metrics from your StatsD servers to Datadog. See [whatever documentation] to learn how to do that.